### ISSUE TITLE
BST Add() must return a node

### ISSUE THREAD

**Issue by mykolachub** [gnomedb/GnomeDB#24]
I guess _add_ method must return a node. That is needed for other manipulation over certain node. As an example _getHeight_ method needs node as parament to work on.
```js
const node1 = tree.add({ _id: 0, name: 'Name', city: 'City' });
const node2 = tree.add({ _id: 1, name: 'Name1', city: 'City1' });
const node3 = tree.add({ _id: 2, name: 'Name2', city: 'City2' });
const node4 = tree.add({ _id: 3, name: 'Name3', city: 'City3' });
```

**Comment by JenyaFTW** [comment]
Alright, will do this in a bit. Busy playing Escape the Gungeon

**Comment by jenyaftw** [comment]
Fixed this in pull request #25, please check it out.

**Comment by mykolachub** [comment]
Thanks, already solved in #26
